specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-07-01/computeRPCommon.json

```diff
@@ -753,10 +753,11 @@
       "properties": {
         "securityEncryptionType": {
           "type": "string",
-          "description": "Specifies the EncryptionType of the managed disk. It is set to DiskWithVMGuestState for encryption of the managed disk along with VMGuestState blob, and VMGuestStateOnly for encryption of just the VMGuestState blob. **Note:** It can be set for only Confidential VMs.",
+          "description": "Specifies the EncryptionType of the managed disk. It is set to DiskWithVMGuestState for encryption of the managed disk along with VMGuestState blob, VMGuestStateOnly for encryption of just the VMGuestState blob, and NonPersistedVMGuestState for not persisting the contents of VMGuestState blob. **Note:** It can be set for only Confidential VMs.",
```

Comment:
it has to be "and NonPersistedVMGuestState for not persisting firmware state in the VMGuestState blob".